dev/app.js

```diff
@@ -654,6 +653,42 @@ renderSiteApp(
                 }
             ]
         },
+        {
+            pathname: "/padding",
+            children: [
```

Comment:
Is it with an intention that all the children have the same title ?

Reply:
In a way, Yes. 
Currently we only have Gantt custom padding, when we will fix the padding issues (#45 and #22) we will need to verify somehow. This is in preparation for it.